actionpack/lib/action_dispatch/system_testing/test_helpers/screenshot_helper.rb

```diff
@@ -2,6 +2,25 @@
 
 module ActionDispatch
   module SystemTesting
+    # Deletes screenshots older than the provided TTL (time to live) in seconds.
+    #
+    # This method is supposed to be called in the beginning of a test suite run and after
+    # test files and configuration have been loaded.
+    def self.rotate_screenshots(ttl:)
+      screenshots_dir = Capybara.save_path.presence || "tmp/screenshots"
```

Comment:
Don't like this duplication; probably, extracting the `screenshots_dir` method to the `SystemTesting` module would make sense.